Makefile

```diff
@@ -32,5 +32,5 @@ tf.destroy:
 	terraform -chdir=terraform destroy
 
 diagram:
-	python ./diagrams/vpc.py
+	python3 ./diagrams/vpc.py
```

Comment:
lol actually python is broken on my machine rn so I can't actually generate an updated image. Sigh...